<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateMapProgressClass</class>
 <widget class="QWidget" name="GenerateMapProgressClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate map</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>541</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deltaTimeLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="numThreadsLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Elapsed time (s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of threads</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>GenerateMapProgressClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>GenerateMapProgressClass</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start()</slot>
 </slots>
</ui>
